src/status_im/ui/components/list_item/views.cljs

```diff
@@ -364,26 +396,51 @@
         subtitle-row-elements
         {:subtitle               subtitle
          :subtitle-max-lines     subtitle-max-lines
-         :subtitle-row-accessory subtitle-row-accessory}]
-    [react/view {:margin-top    container-margin-top
-                 :margin-bottom container-margin-bottom}
-     [react/touchable-highlight
-      (cond-> {:on-press       on-press
-               :on-long-press  on-long-press
-               :underlay-color colors/black
-               :disabled       (or (not on-press) disabled?)}
-        accessibility-label
-        (assoc :accessibility-label accessibility-label))
-      [react/view {:style (styles/container type theme)}
-       (when icon
-         [icon-column icon theme disabled?])
-
-       (when (or title subtitle content)
-         [title-column
-          title-row-elements subtitle-row-elements
-          type icon disabled? theme content accessories])
-
-       (when accessories
-         [accessories-column accessories theme])]]
-     (when error
-       [tooltip/tooltip error styles/error])]))
+         :subtitle-row-accessory subtitle-row-accessory}
+        width (reagent/atom 0)]
+    (reagent/create-class
+     {:reagent-render
+      (fn
+        [{:keys
+          [icon title-prefix title title-row-accessory subtitle
+           subtitle-max-lines subtitle-row-accessory content
+           accessories on-press on-long-press error disabled? selected?]}]
+        (let [title-row-elements
+              (merge title-row-elements
+                     {:title               title
+                      :title-prefix        title-prefix
+                      :title-row-accessory title-row-accessory})
+              subtitle-row-elements
+              {:subtitle               subtitle
+               :subtitle-max-lines     subtitle-max-lines
+               :subtitle-row-accessory subtitle-row-accessory}]
+          ^{:key react-key}
+          (case type
```

Comment:
`if` ?

Reply:
definitely.. I was very very tempted to add `item-description` case, fairly common in unrealized-yet Profile subscreens :), but decided(I think for better) against it.